src/app/map-viewer/map-viewer.service.ts

```diff
@@ -0,0 +1,76 @@
+import { Map } from 'leaflet';
+import { Injectable } from '@angular/core';
+import { Subject } from 'rxjs/Subject';
+import { Observable } from 'rxjs/Rx';
+
+@Injectable()
+export class MapViewerService {
+  public map: Map;
+  public baseMaps: any;
+  public currentLat: number;
+  public currentLng:  number;
+  public altitude: number;
+  public accuracy: number;
+  hasLocation = false;
+  public center: any;
+
+  locationSource = new Subject<any>();
+
+  geoOptions = {
+      enableHighAccuracy: true,
+      maximumAge: 30000,
+      timeout: 27000
+  };
+  constructor() {
+    this.hasLocation = false;
+      if (navigator.geolocation) {
```

Comment:
i don't think that's really core to the "mission" of the about us page. in fact, i think we are trying to accomplish the opposite - displaying coders from *everywhere* in the world, not focusing locally or on any other specific location.